cmd/puppet-db/export.go

```diff
@@ -1,21 +1,60 @@
 package main
 
 import (
+	"errors"
 	"fmt"
+	"os"
 
+	"github.com/puppetlabs/puppetdb-cli/api"
+	app "github.com/puppetlabs/puppetdb-cli/app/puppet-db"
+	"github.com/puppetlabs/puppetdb-cli/log"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var exportCmd = &cobra.Command{
 	Use:   "export",
 	Short: "export <path> [--anon=<profile>]",
-	Run:   executeExportCommand,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("Must specify a single file to write to")
+		}
+		return nil
+	},
+	Run: executeExportCommand,
 }
 
 func init() {
 	rootCmd.AddCommand(exportCmd)
 }
 
 func executeExportCommand(cmd *cobra.Command, args []string) {
-	fmt.Print("not implemented")
+	anonymizationProfile := viper.GetString("anon")
```

Comment:
Should we check that value is one of: full, moderate, low ?

Reply:
I also thought of that, would be another thing to maintain if it changes frequently. I'll add a check though, it doesn't seem likely that these profiles would change often.

Reply:
On second thought the allowed values may not be known in advance, better send it transparently to server